package.json

```diff
@@ -4,69 +4,71 @@
+    "pkg-all-local": "yarn verdaccio-clean && yarn verdaccio-start && yarn verdaccio-connect && yarn publish-to-verdaccio && yarn pkg-all && yarn verdaccio-disconnect",
+    "pkg-all": "bash -c 'source .circleci/local_publish_helpers.sh && generatePkgCli'",
     "pkg-clean": "rimraf build out pkg/node_modules pkg/yarn.lock",
     "prettier-check": "yarn prettier --check .",
     "prettier-changes": "git diff --name-only --diff-filter MRA dev | xargs yarn prettier --write",
     "prettier-write": "yarn prettier --write .",
-    "production-build": "yarn --frozen-lockfile --network-concurrency 1 --cache-folder ~/.cache/yarn && nx run-many --target=build --all",
+    "production-build": "lerna run build --concurrency 3 --stream",
     "promote-rc": "./scripts/promote-rc.sh",
     "publish-to-verdaccio": "lerna publish --yes --no-commit-hooks --no-push --exact --dist-tag=latest --conventional-commits --no-git-tag-version --no-verify-access",
     "release-rc": "./scripts/release-rc.sh",
-    "rm-aa-dev-link": "rimraf -f \"$(yarn global bin)/amplify-app-dev\"",
-    "rm-dev-link": "rimraf -f \"$(yarn global bin)/amplify-dev\"",
-    "setup-dev-win": "yarn dev-build && yarn link-win && yarn link-aa-win",
-    "setup-dev": "yarn dev-build && yarn rm-dev-link && yarn link-dev && yarn rm-aa-dev-link && yarn link-aa-dev",
-    "smoketest": "nx run-many --parallel=1 --target=smoketest --all --output-style=stream-without-prefixes",
-    "smoketest@latest": "CLI_VERSION=latest nx run-many --parallel=1 --target=smoketest --all --output-style=stream-without-prefixes",
-    "smoketest@rc": "CLI_VERSION=rc nx run-many --parallel=1 --target=smoketest --all --output-style=stream-without-prefixes",
+    "rm-aa-dev-link": "rimraf -f \".bin/amplify-app-dev\"",
+    "rm-dev-link": "rimraf -f \".bin/amplify-dev\"",
+    "setup-dev-win": "yarn build && yarn link-win && yarn link-aa-win",
+    "setup-dev": "yarn build && yarn rm-dev-link && yarn link-dev && yarn rm-aa-dev-link && yarn link-aa-dev",
+    "smoketest": "lerna run smoketest --output-style=stream-without-prefixes",
+    "smoketest@latest": "CLI_VERSION=latest lerna run smoketest --output-style=stream-without-prefixes",
+    "smoketest@rc": "CLI_VERSION=rc lerna run smoketest --output-style=stream-without-prefixes",
     "split-e2e-tests": "yarn ts-node ./scripts/split-e2e-tests.ts && git add .circleci/config.yml",
     "split-e2e-tests-codebuild": "yarn ts-node ./scripts/split-e2e-tests-codebuild.ts",
-    "test-changed": "nx affected --target=test",
-    "test-ci": "nx run-many --parallel=4 --target=test --all --exclude='@aws-amplify/amplify-provider-awscloudformation' -- --ci --runInBand && nx run-many --target=test --projects='@aws-amplify/amplify-provider-awscloudformation' -- -ci --runInBand",
-    "test": "nx run-many --target=test --all -- --runInBand",
+    "test-changed": "lerna run test --since dev",
+    "test-ci": "lerna run test --concurrency 4 -- --ci -i",
```

Comment:
we may need to revisit this and implement the exclusion logic for the amplify-provider-awscloudformation test again, but this time with Lerna